src/solver/conv_ocl_dir2D_bwdWrW_2.cpp

```diff
@@ -476,6 +476,9 @@ bool ConvOclBwdWrW2<N_BATCH_LOOPS>::IsApplicableBase(const ConvolutionContext& p
         return false;
     if(!(params.IsFp32() || params.IsFp16() || params.IsBfp16()))
         return false;
+    // Workaround for issue 2534
+    if(params.GetStream().GetDeviceName() == "gfx908")
+        return false;
```

Comment:
[Notice] IIRC this workaround should be removed prior merging.

Reply:
Is it fixed?

Reply:
No. There is workaround PR #400 that seems stuck in Draft state. Let's discuss in https://github.com/AMDComputeLibraries/MLOpen/issues/2534.

Reply:
Please let me know if you need assistance from me.

Reply:
I will check in some new changes, meanwhile I will remove this workaround. If the issue is still there, I will request your help.

Reply:
done